core/SiteContentDetector.php

```diff
@@ -78,55 +88,82 @@ private function resetDetectionProperties() : void
      *                                   will speed up the detection check
      * @param ?int        $idSite        Override the site ID, will use the site from the current request if null
      * @param string|null $siteData      String containing the site data to search, if blank then data will be retrieved
-     *                                   from the current request site via cURL
+     *                                   from the current request site via an http request
      * @param int         $timeOut       How long to wait for the site to response, defaults to 60 seconds
+     * @return void
      */
     public function detectContent(array $detectContent = [SiteContentDetector::ALL_CONTENT],
-                                  ?int $idSite = null, ?string $siteData = null, int $timeOut = 60)
+                                  ?int $idSite = null, ?string $siteData = null, int $timeOut = 60): void
```

Comment:
is `60` a bit long for the timeout? Maybe we could set this to the same as the api.matomo.org (I think `2`) or plugins.matomo.org calls?

Reply:
60 seconds is too long, I agree :+1: 
API calls are going to be a bit quicker than a typical site main page load though which might be served by a slow, bloated CMS. Would 5 seconds be acceptable?

Reply:
@mattab do you have any preference as PM on the timeout for checking this?

Reply:
have just confirmed with @mattab that 5s is acceptable for now. Let's proceed @bx80

Reply:
That's great, I already changed the default timeout to 5s with the last few changes :+1: